### DISCUSSION TITLE
Please sole this

### DISCUSSION THREAD

**Discussion by mishrapcdac** [Security-Onion-Solutions/securityonion#5278] [Unsupported Versions]
my system is hanged while installing security onion standalone setup why?? after completion of 66% it always hangs

this package will be installed using the internet or offline

**Comment by dougburks** [comment]
First, check the Hardware Requirements:
https://docs.securityonion.net/en/2.3/hardware.html

If that doesn't help, then you'll need to provide more information as described at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720.

**Reply by mishrapcdac** [reply]
after running so-status command it shows all the services in waiting state

then tell me whether it is installed or not?

**Reply by mishrapcdac** [reply]
please help me...what should i do....

security onion is installed in the  base machine

**Reply by gcsrellis** [reply]
Have you verified your hardware specs are enough to run Security Onion?  How many processors do you have?  How much RAM do you have?  These can all effect how the install goes.
